### ISSUE TITLE
Register for push is not setting token

### ISSUE THREAD

**Issue by cdmunoz** [la-haus/iterable-flutter#1]
After initialize and setting the email or setUserId, `registerForPush` is not updating the device token for the user.

```dart
    "initialize" -> {
        val apiKey = call.argument<String>("apiKey") ?: ""
        val pushIntegrationName = call.argument<String>("pushIntegrationName") ?: ""

        if (apiKey.isNotEmpty() && pushIntegrationName.isNotEmpty()) {
          initialize(apiKey, pushIntegrationName)
        }
        result.success(null)
      }
...
    "setEmail" -> {
        val userEmail = call.arguments as String
        IterableApi.getInstance().setEmail(userEmail)
        IterableApi.getInstance().registerForPush()
        result.success(null)
      }
...
private fun initialize(apiKey: String, pushIntegrationName: String) {
    val config = IterableConfig.Builder()
        .setLogLevel(Log.DEBUG)
        .setPushIntegrationName(pushIntegrationName)
        .setAutoPushRegistration(false)
        .build()
    IterableApi.initialize(context, apiKey, config)
  }
```

In order to have a working sample, we have been manually getting the device token as follows:

```kotlin
FirebaseMessaging.getInstance().token
        .addOnCompleteListener(OnCompleteListener { task ->
          if (!task.isSuccessful) {
            Log.e("initialize error >>>", "Fetching FCM registration token failed", task.exception)
            return@OnCompleteListener
          }

          // Get new FCM registration token
          val token = task.result

          Log.e("initialize token >>>", "Fetching FCM registration token $token")
        })
```
Then calling API directly:
[https://api.iterable.com/api/docs#users_registerDeviceToken](https://api.iterable.com/api/docs#users_registerDeviceToken)

**Comment by danielgomezrico** [comment]
@cdmunoz did we already fixed this on https://github.com/la-haus/iterable-flutter/releases/tag/v0.1?

**Comment by cdmunoz** [comment]
Yes, we did! thanks for the heads up! 
Closing it with fix in [v0.1](https://github.com/la-haus/iterable-flutter/releases/tag/v0.1)
